api/numaresourcesoperator/v1alpha1/numaresourcesoperator_types.go

```diff
@@ -57,12 +59,22 @@ type NodeGroup struct {
 
 // NUMAResourcesOperatorStatus defines the observed state of NUMAResourcesOperator
 type NUMAResourcesOperatorStatus struct {
-	DaemonSets []NamespacedName `json:"daemonsets,omitempty"`
-
+	DaemonSets         []NamespacedName    `json:"daemonsets,omitempty"`
+	MachineConfigPools []MachineConfigPool `json:"machineconfigpools,omitempty"`
```

Comment:
next time please elaborate in the commit message *why* we do like this, which problem we want to solve/mitigate and in general everything that will allow us our future selves in 6 months for now to reconstruct our thinking now.